### DISCUSSION TITLE
Mediation should be automated

### DISCUSSION THREAD

**Discussion by Wolfstorm321** [bisq-network/bisq#6941] [General]
I'm new to BISQ (started now in October) and never had any issue with trades. However, two days ago a trade have gone wrong, and I didn't receive the payment. 

It has been 50+ hours since that trade started. The buyer refused to pay, and now my bitcoin is stuck in the trade. 
I called the mediator, he said the buyer dropped out the trade and is going to pay a penalty. 
10+ hours have passed and nothing. I ask the mediator how much time until I receive my btc back, he don't answer. 

I tried to use the matrix.io indicated in the software. Tried four different apps, none of them work, and there is no sign up procedure. I don't have a clue how matrix.io works, it looks extremely complicated. So I'm asking this here. How much time until the btc is unlocked? And why it takes so long? 

I think mediation should be automated. If payment is not made, the bitcoin is returned to the seller, and mining fee is paid by the buyer, since he dropped the trade.

**Comment by MwithM** [comment]
Hi, read the [dispute resolution](https://bisq.wiki/Dispute_resolution) wiki.
Basically, if your peer does not reply or accept mediator suggestion, you'll need to wait 10 days (altcoin) or 20 (fiat) to reject mediation suggestion, which will spend the funds in the 2of2 multisig, which is required to be reimbursed in arbitration. 
Bisq was built for fiat/btc trades and altcoin trading was added because it was easy to add later. Automating a fiat trade is not possible, and the only pair where something similar to auto trading is the auto confirmation feature for Monero, which anyways is not mandatory and sometimes fails, so traders sometimes still need mediation for those.

**Comment by pazza83** [comment]
There is also a proposal to allow the responsive trader to be able to claim funds prior to arbitration that will allow trader more autonomy with the dispute process:

https://github.com/bisq-network/proposals/issues/421

It is hard to automate the mediation aspect due to fiat trades needing proof of funds being sent / received etc.
